app/main.ipynb

```diff
@@ -192,18 +244,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
-   "outputs": [],
-   "source": [
-    "# write your code here"
-   ],
+   "source": "column_mean = np.mean(A, axis=1)",
```

Comment:
The mean is calculated along axis 1 (rows), but the task requires the mean for each column (axis 0). Change 'axis=1' to 'axis=0' to calculate the mean for each column.